### PR TITLE
automating docs generate

### PR DESCRIPTION
**Pull request by fivetran-sheringuyen**
**What change does this PR introduce?** 
 
Adds new macro to automate docs generate part of package workflow

**If this PR introduces a new macro, how did you test the new macro?** 

Copied an existing package and ran the following tests from adjacent development directory and installed local copy of dbt fivetran utils:
1. Updating docs: Replaces existing project/docs contents with new docs
2. Creating docs: Creates new docs directory and moves integration_tests/target/ docs files to new docs folder

**If this PR introduces a modification to an existing macro, which packages is the macro currently present in and what steps were taken to test compatibility across packages?** 

New macro

**Did you update the README to reflect the macro addition/modifications?** 

- [x] Yes
- [ ] No (provide further explanation)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -578,6 +600,21 @@ source dbt_packages/fivetran_utils/generate_columns.sh '../dbt_apple_search_ads_
 In that example, it will:
 * Create a `get_campaign_history_columns.sql` file in the `macros` directory, with the necessary macro within it.
 
+----
+
+### generate_docs.sh([source](generate_docs.sh))
+
+This bash file can be used to create or replace package documentation (`<project_name>/docs`).
```
Comment (fivetran-joemarkiewicz) [inline README.md:607]: It isn't clear where I need to run this command. Do I run it in the direction of the `integration_tests` folder, or within the root of my `development` dbt project? Can we specify that here.

File: generate_docs.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+ORIG_DIR=$(echo $PWD)
+PKG=$(echo $1)
+cd ${PKG}/integration_tests
+dbt clean && dbt deps && dbt seed && dbt run
```
Comment (fivetran-joemarkiewicz) [inline generate_docs.sh:5]: Do we need to run `dbt run`? Could we limit the steps by just executing `dbt clean && dbt deps && dbt seed && dbt docs generate`?

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
This is super cool @fivetran-sheringuyen!

I am running into a bit of a blocker on getting it to work however. Please see my inline comments. Additionally, below is the result I get when running the source command.
image

The end result then removes the files from my `/docs` folder, but then doesn't regenerate them?

File: README.md (hunk 1 above)
Comment (fivetran-sheringuyen) [reply README.md:607]: Sure thing! Where is your development folder located wrt the dbt_jira_source directory?

Comment (fivetran-sheringuyen) [reply README.md:607]: image
This macro is designed to work in all directories under `stripe` (the project directory) that exist at the same level as `dbt_stripe`. So you can run this from `dev` which exists adjacent to `dbt_stripe`. I hope that helps! If it still doesn't work let me know. 

@fivetran-joemarkiewicz

**Review by fivetran-joemarkiewicz — APPROVED** [review]
Looks great! 

One thing to add before merging is highlighting the fact that we need to update our target schema in `profiles.yml`

**Pull request merged**